src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/prestashop_ui_kit_base.html.twig

```diff
@@ -1161,7 +1161,7 @@
 
 {% block form_help %}
   {% if help is not null %}
-    <small class="form-text">{{ help|raw }}</small>
+    <small class="form-text {% if help_attr.class is defined %}{{ help_attr.class }}{% endif %}" {% if help_attr.placeholder is defined %} data-place-holder="{{ help_attr.placeholder }}" {% endif%}>{{ help|raw }}</small>
```

Comment:
You can probably use similar code to the original Symfony form theme for this params:

https://github.com/symfony/symfony/blob/4.4/src/Symfony/Bridge/Twig/Resources/views/Form/form_div_layout.html.twig#L305-L306

Except we don't need the default class maybe, oh actually we do have one!! `form-text`